tests/core/test_stddef.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#ifdef __cplusplus
+#include <cstddef>
+using namespace std;
+#else
+#include <stddef.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Declare puts manually, we don't want to include any other headers here
```

Comment:
Why do we even need the puts? Can't just the exit status (i.e. it didn't abort) be enough to say the test passed?

Reply:
Its just the way these "in/out" tests are currently setup.   I have a plan to clean this up, but for not we need and out file to exist.  I guess it could be empty?  In any case there are several tests following this pattern already and we should clean them up all at once.

Reply:
OK, sounds good.